docs/build-the-docs.md

```diff
@@ -35,6 +35,33 @@ antora site.yml
 
 If all goes well, you will _not_ see any console output.
 
+#### Testing For Broken Xref Links
+
+There are a host of ways to check for broken links, including NPM's broken link checker.
+However, this custom generator, provided by the core Antora team, is able to check [the native Antora xref links](https://docs.antora.org/antora/1.0/asciidoc/page-to-page-xref/#xref-and-page-id-anatomy).
+It doesn't check external links, but it's still a good start.
+To use it, you need to pass the custom generator (in `./generator/xref-validator`) to the `generate` command, as in the example below.
+```console
+antora generate \
+    --pull \
+    --ui-bundle-url https://github.com/owncloud/docs-ui/releases/download/1.1.0/ui-bundle.zip \
+    --generator=./generator/xref-validator \
+    site.yml
```

Comment:
``site.yml`` --> ``site.prod.yml``

Reply:
That would have to happen later, as that file's not part of this branch. The docs can be updated later.